caproto/_cli.py

```diff
@@ -680,7 +680,7 @@ def put(pv_name, data, *, verbose=False, timeout=1, priority=0, repeater=True):
             for command in commands:
                 if (isinstance(command, ca.WriteNotifyResponse) and
                             command.ioid == req.ioid):
-                    response = command
+                    # response = command
```

Comment:
why?  Looks like it isn't used?

Reply:
@danielballan ?

Reply:
Relic of a copy/paste I think. Safe to delete.